From d32617aaf89437df8cbf577acad6cd072f7a441f Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Wed, 15 Nov 2023 15:31:04 +0100
Subject: [PATCH] arm64: dts: st: remove dmas from usart2 serial console on
 stm32mp257f-ev1

Don't use DMA for serial console, so remove dmas and dma-names properties
from usart2 which is used for serial console on stm32mp257f-ev1.

Change-Id: I6bd366bf3e69bd077a7c986eff3b94eeda2a7861
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 7ceffcbc9a29..4c7c7fa5fcc7 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -238,6 +238,8 @@ &usart2 {
 	pinctrl-0 = <&usart2_pins_a>;
 	pinctrl-1 = <&usart2_idle_pins_a>;
 	pinctrl-2 = <&usart2_sleep_pins_a>;
+	/delete-property/dmas;
+	/delete-property/dma-names;
 	status = "okay";
 };
 
-- 
2.39.5

